src/Desktop/UIAutomation/CustomObjects/Converters/PointTypeConverter.cs

```diff
@@ -0,0 +1,17 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+
+namespace Axe.Windows.Desktop.UIAutomation.CustomObjects.Converters
+{
+    class PointTypeConverter : ITypeConverter
+    {
+        public string Render(dynamic value)
+        {
+            if (value == null) throw new ArgumentNullException(nameof(value));
+            double[] arr = (double[])value;
+            return $"[x={arr[0]},y={arr[1]}]";
```

Comment:
Strictly speaking, this is returning an [interpolated string](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/tokens/interpolated), which is subject to the user's culture setting. This might lead to an issue in TSA (a build-time analysis database). It's been so long since we had one of these warnings (we fixed them all) that I'm not 100% sure that this still shows up as a warning. Let's try it for now, then we can add the trivial change to fix it later if we get a TSA warning.